gamersnet_backend/routes/posts/index.js

```diff
@@ -1,12 +1,17 @@
 const router = require('express').Router();
 
 // include each route handler
+let { removePost } = require('./deletePost');
 let {createPost} = require('./createPost');
-let {listAllPosts, listValidPosts} = require('./getPosts');
+let {listAllPosts, listValidPosts, listUserPosts} = require('./getPosts');
 
 router.post('/createPost', createPost);
 router.get('/listAllPosts', listAllPosts);
 router.get('/listValidPosts', listValidPosts);
+router.delete('/deletePost', removePost);
+router.post('/listUserPosts', listUserPosts);
```

Comment:
This is related to the comment about the user ID above (POST vs GET?)

Reply:
yes, will update it to GET!